daemon/mgr/image.go

```diff
@@ -89,3 +92,14 @@ func (mgr *ImageManager) SearchImages(ctx context.Context, name string, registry
 
 	return nil, nil
 }
+
+// ImageRemoveOption wraps the image remove interface params.
+type ImageRemoveOption struct {
+	Force bool
+}
```

Comment:
I think we can create the "image_types.go" file, put the "ImageRemoveOption" into it. 
WDYT ????